inbound-mailer-system/src/index.ts

```diff
@@ -432,15 +433,33 @@ export class InboundMailer {
     return { data, error };
   }
 
-  async updateInboundTextAsUnprocessed(user_id: string, text_ids: string[]) {
```

Comment:
There is also an `updateInboundEmailAsUnprocessed` which we might need to do something with, however, it's currently in use here: https://github.com/CodeKarateStudios/audiofeed/blob/c8805325c7ae8c4b0d69d58dd92d4965805bb33d/src/utils/generators/episode.ts#L67

This seems to tie into the plan limits (which don't appear to be fully working anyway right now).

Reply:
Actually, based on reading your other comment. I think what we want to do is have "unprocessed" mean that it wasn't processed because of plan limits while "cancelled" means the user cancelled it from the UI.